src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -180,22 +179,16 @@ const Actions: React.FC<Props> = ({
                     onClose={() => setActionAnchorElement(null)}
                     anchorEl={actionAnchorElement}
                 >
-                    <MenuList id="action-menu">
-                        {availableActionsForUserRole.map(({ actionName }) => {
-                            return (
-                                <MenuItem
-                                    key={actionName}
-                                    onClick={onMenuItemClick(
-                                        actionName,
-                                        isNotAtLeastOne,
-                                        errorMessage
-                                    )}
-                                >
-                                    {actionName}
-                                </MenuItem>
-                            );
-                        })}
-                    </MenuList>
+                    {availableActionsForUserRole.map(({ actionName }) => {
+                        return (
+                            <MenuItem
```

Comment:
Is it possible to have the 'Change Packing Date' action highlighted upon open of actions? while pressing enter does perform this action, it may be clearer for the user if the actual action is selected before they press any keys. 
Same for 'No Status' on the Statuses component .